portfolio.html

```diff
@@ -334,6 +334,7 @@ <h3>LET'S CONNECT</h3>
           Get It Touch
         </button>
       </form>
+      <div class="bottomline"></div>
```

Comment:
Please change `Contact me` to be any heading from `h2` to `h6`, for example: `h2` instead of `div` tag for semantic purposes.